app/views/docs/rest.phtml

```diff
@@ -1,12 +1,331 @@
+    </tr>
+    </tbody>
+</table>
+
+<h3>Permission Roles</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Role String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Role.any()</code></td>
+        <td><code>"any"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.guests()</code></td>
+        <td><code>"guests"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.users([STATUS])</code></td>
+        <td><code>"users/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.user([USER_ID], [STATUS])</td>
+        <td><code>"user:[USER_ID]/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID])</td>
+        <td><code>"team:[TEAM_ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID], [ROLE])</td>
+        <td><code>"team:[TEAM_ID]/[ROLE]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.member([MEMBERSHIP_ID])</td>
+        <td><code>"member:[MEMBERSHIP_ID]"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<ul>
+    <li class="margin-bottom"><a href="/docs/permissions" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Learn more about permissions</a></li>
+</ul>
+
+<h2>Unique ID</h2>
+<p>
+Appwrite's SDKs have a helper <code>ID.unqiue()</code> to generate unique IDs. When using Appwrite without an SDK, pass the string <code>"unique()"</code> into the ID parameter.
+</p>
+
+<h2>Query Methods</h2>
+<p>
+Appwrite's SDKs provides a <code>Query</code> class to generate query strings. When using Appwrite without an SDK, you can template your own strings with the format below.
+</p>
+
+<p>Query strings are passed to Appwrite using the <code>queries</code> query parameters. You can attack multiple query strings under the <code>queries</code> query parameter.</p>
+
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 150px">Query Method</th>
+            <th>Query String</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>equal</td>
+            <td><code>equal("attribute", ["value1", ...])</code></td>
+        </tr>
+        <tr>
+            <td>notEqual</td>
+            <td><code>notEqual("attribute", ["value1", ...])</code></td>
+        </tr>
+        <tr>
+            <td>lessThan</td>
+            <td><code>lessThan("attribute", [value1, ...])</code></td>
+        </tr>
+        <tr>
+            <td>lessThanEqual</td>
+            <td><code>lessThanEqual("attribute", [value1, ...])</code></td>
+        </tr>
+        <tr>
+            <td>greaterThan</td>
+            <td><code>greaterThan("attribute", [value1, ...])</code></td>
+        </tr>
+        <tr>
+            <td>greaterThanEqual</td>
+            <td><code>greaterThanEqual("attribute", [value1, ...])</code></td>
```

Comment:
We should probably discourage passing multiple values for all of these because it doesn't really make sense.

Reply:
I'll add a warning. Since we can, we should document it as such but let them know